configure.ac

```diff
@@ -2686,8 +2686,8 @@ AC_SUBST(SKIP_PERL5)
 
 
 SKIP_PYTHON=
-if (test -z "$PYINCLUDE" || test -z "$PYLINK") &&
-   (test -z "$PY3INCLUDE" || test -z "$PY3LINK") ; then
```

Comment:
What's the point of adding in a redundant semicolon and making a whitespace change (we tend not to put in this kind of whitespace)?

Reply:
oh sorry, ignore, it's not the same bracket